Update upload private linking admonition (#299)
neo4j/docs-aura: modules/ROOT/partials/import-database.adoc

[CAUTION]
====
If you have enabled xref:platform/security.adoc#_vpc_isolation[private linking], you must ensure public traffic is enabled for the region your instance is hosted in before you can use this command.

> Public traffic can be enabled from the *Network access* section of the Console by selecting *Configure* on the region you want to change and un-ticking the *Disable public access* option in step 4.

Reviewer: Maybe "deselecting" is better than "un-ticking"?

Author: Microsoft suggests 'clear' but i wonder if that is even more confusing - https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/deselect

Reviewer: Same for "unchecking": https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/uncheck-unmark-unselect. I agree it may be more confusing too.

How about rephrasing as the following:

```suggestion
You can enable public traffic from the *Network access* section of the Console by selecting *Configure* on the region you want to change and making sure the *Disable public access* option in step 4 is not selected.
```
